ru.sview.sView.json

```diff
@@ -51,8 +51,8 @@
 	    "sources": [
 		{
 		    "type": "archive",
-		    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
```

Comment:
@hadess , could you comment for those unfamiliar with flatpack - what for wxWidgets is used for?

Reply:
Has nothing to do with Flatpak, it was a dependency of codeblocks.